content/ja/_index.html

```diff
@@ -41,12 +41,12 @@ <h2>150以上のマイクロサービスアプリケーションをKubernetes上
         <button id="desktopShowVideoButton" onclick="kub.showVideo()">ビデオを見る</button>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccncna20" button id="desktopKCButton">2020年11月17日-20日のKubeCon NAバーチャルに参加する</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccncna22" button id="desktopKCButton">2022年10月24日から28日までミシガン州のKubeConデトロイトに参加する</a>
```

Comment:
```suggestion
        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccnceu22" button id="desktopKCButton">2022年5月16日〜20日のKubeCon EU バーチャルに参加する</a>
```

Reply:
To keep consistency with the previous events.

https://kubernetes.io/ja/

Reply:
Updated